plugins/vpp/ifplugin/nat_config.go

```diff
@@ -182,7 +182,7 @@ func (plugin *NatConfigurator) SetNatGlobalConfig(config *nat.Nat44Global) error
 	plugin.globalNAT = config
 
 	// Forwarding
-	if err := plugin.natHandler.SetNat44Forwarding(config.Forwarding); err != nil {
+	if err := plugin.natHandler.SetForwarding(config.Forwarding); err != nil {
```

Comment:
I see that currently there is no `Nat64ForwardingEnableDisable`, but is there possibility it might ever collide with some new binapi for enabling forwarding not related to *NAT44*?